drivers/staging/vc04_services/interface/vchiq_arm/vchiq_core.c

```diff
@@ -2097,7 +2097,6 @@ slot_handler_func(void *v)
 					vchiq_log_error(vchiq_core_log_level,
 						"Failed to send RESUME "
 						"message");
```

Comment:
We don't use suspend and resume on Raspberry Pi, so either way it is OK to delete this BUG.